index.html

```diff
@@ -3990,30 +3954,21 @@ <h2>
             </td>
             <td>
               <ul>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-readonly">`aria-readonly`</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-required">`aria-required`</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-selected">`aria-selected` (state)</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-expanded">`aria-expanded` (state)</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-colspan">`aria-colspan`</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-colindex">`aria-colindex`</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-rowindex">`aria-rowindex`</a>
-                </li>
-                <li>
-                  <a data-cite="wai-aria-1.1#aria-rowspan">`aria-rowspan`</a>
-                </li>
+                <li><a data-cite="wai-aria-1.1#aria-colindex">`aria-colindex`</a></li>
+                <li><a data-cite="wai-aria-1.2#aria-colindextext">`aria-colindextext`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-colspan">`aria-colspan`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-disabled">`aria-disabled`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-errormessage">`aria-errormessage`</a></li>
+                <li><a data-cite="wai-aria-1.1#aria-expanded">`aria-expanded` (state)</a></li>
```

Comment:
`aria-expanded`, `aria-readonly`, and `aria-required` should only be allowed if NOT a child of a `table`.  so we should separate these out like they were for `row`